lib/base-controller.js

```diff
@@ -4,8 +4,9 @@ var Controller = require('hmpo-form-wizard').Controller;
 var util = require('util');
 var _ = require('lodash');
 
-var BaseController = function BaseController() {
-  this.confirmStep = '/confirm';
+var BaseController = function BaseController(options) {
+  this.options = options || {};
```

Comment:
Pretty sure this is redundant. It will be clobbered by one of the parent constructors.
I'd suggest just referencing the objects argument directly in the next line.